modules/shibboleth/templates/shibboleth2.xml.erb

```diff
@@ -65,6 +65,14 @@
                     <md:GivenName>Unity SSO Support</md:GivenName>
                     <md:EmailAddress>unity-sso@researchresearch.com</md:EmailAddress>
                 </md:ContactPerson>
+
+                <mdui:UIInfo>
+                    <mdui:DisplayName xml:lang="en">*Unity</mdui:DisplayName>
+                    <mdui:Description xml:lang="en">*Unity is the new way to work with anyone in universities and research. It’s a simple place to form groups, share files and work on them together.</mdui:Description>
+                    <mdui:InformationURL xml:lang="en">http://www.unity.ac</mdui:InformationURL>
+                    <mdui:Logo height="60" width="80">https://shib-sp.unity.ac/assets/unity-sso.png</mdui:Logo>
+                </mdui:UIInfo>
```

Comment:
It wouldn't be valid XML and could cause problems either immediately or in the future. The docs seem to be full of XML syntax typos and oversight -- I wouldn't trust them in a literal sense.

That said, from those docs (and then forgotten later):

```
Additionally, make sure the md and mdui xml namespaces are defined in the <SPConfig> tag on the top of shibboleth.xml:

<SPConfig xmlns="urn:mace:shibboleth:2.0:native:sp:config"
    xmlns:conf="urn:mace:shibboleth:2.0:native:sp:config"
    xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion"
    xmlns:samlp="urn:oasis:names:tc:SAML:2.0:protocol"
    xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata"
    xmlns:mdui="urn:oasis:names:tc:SAML:metadata:ui"
    clockSkew="180">
```